Library/Homebrew/extend/ARGV.rb

```diff
@@ -11,9 +11,9 @@ def flags_only
     select { |arg| arg.start_with?("--") }
   end
 
-  def formulae
+  def formulae(is_installing = false, warn_all_ambiguity = false)
```

Comment:
This should instead take an `options` hash and use the current names as hash keys. This will improve readability of callers.